lib/pages/other/WebView.dart

```diff
@@ -7,37 +9,70 @@ import 'package:app/assets/components/customAppBar.dart';
 import 'package:stacked/stacked.dart';
 import 'package:app/viewmodels/web_view_model.dart';
 
-class WebViewArgumnets {
+class WebViewArguments {
   final String initialUrl;
 
-  WebViewArgumnets(this.initialUrl);
+  WebViewArguments(this.initialUrl);
 }
 
-class WebViewPage extends StatelessWidget {
-  final WebViewArgumnets args;
+class WebViewPage extends StatefulWidget {
+  final WebViewArguments args;
   WebViewPage(this.args);
+  
+
+  @override
+  _WebViewPageState createState() => _WebViewPageState();
+}
+
+class _WebViewPageState extends State<WebViewPage> {
+
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: CustomAppBar(
+      appBar: customAppBar(
         backButtonText: "Back",
         context: context,
-      ),
-      body: SafeArea(
-        child: ViewModelBuilder<WebViewViewModel>.reactive(
-            viewModelBuilder: () => WebViewViewModel(),
-            builder: (context, model, child) {
-              return WebView(
-                javascriptMode: JavascriptMode.unrestricted,
-                initialUrl: args.initialUrl,
-                onWebResourceError: (error) {
-                  model.onWebError(args.initialUrl);
-                },
+        actions: [
+         Padding(
+           padding: const EdgeInsets.all(12.0),
+           child: DropdownButton<String>(
+             iconEnabledColor: Theme.of(context).primaryColor,
+             isDense: true,
+             items: <String>['Open'].map((String stringValue) {
+                 return DropdownMenuItem<String>(
+                   value: stringValue,
+                   child: CustomTextButton(
+              stringValue,
+              onClick: (){
+                NavigationService()
```

Comment:
In general, we should avoid using services directly in views. I think for the NavigationService thought I don't think it really matters so feel free to keep that in here (instead of having to make a viewmodel).

However, to get the navigation service we should use the locator, for example: 

```
final NavigationService _navigationService = locator<NavigationService>();
```

Reply:
Thanks.I'll update it.